app/views/posts/index.html.erb

```diff
@@ -18,7 +18,7 @@
 		<div id="posts">
 			<%= render :partial => 'post', :collection => @posts %>
 		</div>
-
+        <%= link_to 'Flag for Review', '#', data: {toggle: "modal", target: user_signed_in? ? "#flag-modal" : "#login-modal"} %>
```

Comment:
This should actually go into the partial since we are going to allow flagging of individual posts.